{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -34,6 +34,4 @@ RUN set -x \
     && rm -rf /root/.cache/pip \
     && apk del .build-deps
 
-COPY . "${CWD}/"
```

Comment:
Change this to

```dockerfile
COPY src/ tests/ scripts/ gunicorn.py setup.cfg "${CWD}/"
```

or too verbose? Also a little annoying because those targets might change I guess and should be handled by `.dockerignore`.